src/common/commonutils/FileUtils.c

```diff
@@ -295,6 +302,12 @@ static int CheckAccess(bool directory, const char* name, int desiredOwnerId, int
     {
         OsConfigLogInfo(log, "CheckAccess: '%s' not found, nothing to check", name);
         result = 0;
+
+        if (reason)
+        {
+            FREE_MEMORY(*reason);
+            *reason = FormatAllocateString("PASS'%s' not found, nothing to check", name);

```

Comment:
missing space `PASS '%s'`

Reply:
It's by design. There's the prefix for "PASS". There is no space after and "PASS" is cut and not reported as reason. If there would be a space, that would be a bug (as that reason will be prefixed with a blank)

Reply:
See the PR description also.

Reply:
Actually, you found something here. "PASS" written by itself. Will fix this.

Reply:
Fixed. Please see last iteration